libs/ledger-live-common/src/families/cardano/staking.ts

```diff
@@ -0,0 +1,3 @@
+export type { APIGetPoolsDetail, StakePool } from "@ledgerhq/coin-cardano/api/api-types";
+export { fetchPoolDetails } from "@ledgerhq/coin-cardano/api/getPools";
+export { LEDGER_POOL_IDS } from "@ledgerhq/coin-cardano/utils";
```

Comment:
Is this necessary ? Can't UI specific to Cardano simply get the types from `@ledgerhq/coin-cardano` directly ? Or do you specifically want that to be an entrypoint for those types & constants ?

Reply:
I prefer to separate UI from CoinModules direct use. This is why I keep in LLC some entrypoint to access the coin-modules.
Ideally, the logic required by the UI should be LLC. The UI shouldn't have any clue about what the meaning of what it displays.